<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<node pkg="navigation_light" type="record_waypoints.py" name="record_waypoints">
		<param name="filepath"    value="/home/pluto/maps/waypoints_record.csv" />
		<param name="map_frame_id"    value="map" />
		<param name="base_frame_id"    value="base_link" />
		<param name="new_waypoint_thr"    value="0.3" />
	</node>
</launch> 
